typescript/sdk/src/utils/ism.ts

```diff
@@ -0,0 +1,6 @@
+import multisigIsmVerifyCosts from '../consts/multisigIsmVerifyCosts.json';
+
+export function multisigIsmVerificationCost(m: number, n: number): number {
+  // @ts-ignore
+  return multisigIsmVerifyCosts[`${n}`][`${m}`];
```

Comment:
Might be good to have some runtime checks here

Reply:
What would you suggest?

Reply:
like `if !key in obj throw`

Reply:
Yeah  this ^ , with a good error msg in the thrown error